lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/TrashFactory.java

```diff
@@ -0,0 +1,28 @@
+package org.enso.desktopenvironment;
+
+final class TrashFactory {
+
+  private static final class LazyTrash {
+
+    private static Trash Instance = null;
```

Comment:
This is a coding non-sense. Not sure what's the goal, but to get a _lazy trash_ initialization, one can make the field `final` and just initialize it in static initializer.

Reply:
Corrected in 633f78ab22